src/vs/workbench/browser/parts/editor/editorPart.ts

```diff
@@ -460,6 +460,28 @@ export class EditorPart extends Part implements IEditorGroupsService, IEditorGro
 		}
 	}
 
+	getLayout(): EditorGroupLayout {
```

Comment:
There are lots of warnings with this change?

Reply:
Where do you see those?

Reply:
I had just applied your change to my sources locally.